R/pkg/R/generics.R

```diff
@@ -1403,20 +1416,25 @@ setGeneric("unix_timestamp", function(x, format) { standardGeneric("unix_timesta
 #' @export
 setGeneric("upper", function(x) { standardGeneric("upper") })
 
-#' @rdname var
+#' @rdname column_aggregate_functions
+#' @param y,na.rm,use currently not used.
```

Comment:
should this be moved like this line https://github.com/apache/spark/pull/18025/files#diff-8e3d61ff66c9ffcd6ffb7a8eedc08409L923